lib/models/project.js

```diff
@@ -134,7 +134,7 @@ class Project {
     }
     for (let depName in this.dependencies()) {
       try {
-        this.resolveSync(depName);
+        this.resolveSync(depName + '/package.json');
```

Comment:
The manual string concat here is failing linting / CI:

```

not ok 10 eslint should have no errors in lib
  Error: Code did not pass lint rules
  lib/models/project.js
    137:26  error  Unexpected string concatenation  prefer-template
```

Reply:
Thanks. I saw the failure yesterday, but I didn't get a chance to push a fix. It should be good now. 

Some tests failed locally, but I'm not sure if those are related, I'll keep an eye on travis.